tutorials/shaders/compute_shaders.rst

```diff
@@ -30,7 +30,9 @@ So now lets work with a compute shader to see how it really works.
 Creating a ComputeShader
 ------------------------
 
-To begin using compute shaders, create a new text file called "compute_example.glsl". When you write compute shaders in Godot, you write them in GLSL directly. The Godot shader language is based off of GLSL so if you are familiar with normal shaders in Godot the syntax below will look somewhat familiar.
+To begin using compute shaders, ensure you are using a Vulkan based renderer (the Forward+ or Mobile options), or your editor may crash.
```

Comment:
```suggestion
To begin using compute shaders, ensure you are using a RenderingDevice based renderer (the Forward+ or Mobile options) as compute shaders are not available in the GL_Compatibility renderer.
```

Reply:
Got it, should I edit and force push to make this a single commit?

Reply:
That's perfect, ya